teachertool/src/components/HomeScreen.tsx

```diff
@@ -53,6 +58,60 @@ const Card: React.FC<CardProps> = ({ title, className, icon, onClick }) => {
     );
 };
 
+interface LoadingCardProps {
+    delay?: boolean;
+}
+
+const LoadingCard: React.FC<LoadingCardProps> = ({ delay }) => {
+    return (
+        <div className={css.cardContainer}>
+            <Button
+                className={classList(css.cardButton, css.loadingGradient, delay ? css.loadingGradientDelay : undefined)}
+                title={""}
+                onClick={() => {}}
+            >
+                <div className={css.cardDiv}>
+                    <div className={css.loadingGradient}></div>
+                </div>
+            </Button>
+        </div>
+    );
+};
+
+interface RubricResourceCardProps {
+    cardTitle: string;
+    imageUrl: string;
+    rubricUrl: string;
+}
+
+const RubricResourceCard: React.FC<RubricResourceCardProps> = ({ cardTitle, imageUrl, rubricUrl }) => {
+    const onRubricClickedAsync = async () => {
```

Comment:
I feel like this function can be renamed. I find it a bit confusing to read `onRubricClicked` when the user really isn't clicking on a rubric, but on the card to load a rubric. Being that the function is in the `RubricResourceCard` component, can it just be `onCardClicked`?